config/metrics.js

```diff
@@ -133,8 +133,13 @@ module.exports = {
 		extract: (item) => !!getBPRules(item)?.requiresConversationResolution,
 		permissions: ['ADMIN'],
 	},
+	codeOfConduct: {
+
+		extract: (item )=> item.codeOfConduct !=null,
```

Comment:
```suggestion
		extract: (item) => item.codeOfConduct != null,
```

Reply:
altho should this maybe just read the `codeOfConduct` value, so repo-report can check *which* CoC it's using?